modules/Bio/EnsEMBL/Production/Pipeline/Metadata/PayloadGenerator.pm

```diff
@@ -40,6 +40,7 @@ sub run {
     my $database_uri = 'mysql://'.$user.':'.$password.'@'.$host.':'.$port.'/'.$dbname;
     my $email = $self->param_required('email');
     my $metadata_uri = $self->param_required('metadata_uri');
+    my $taxonomy_uri = $self->param_required('taxonomy_uri');
```

Comment:
As mentioned during our meeting, we do not plan to add any new parameter to the meta data updater main request/ This parameter should be a global parameter set when initializing the pipeline

Reply:
Resolved. It is now a global paramater